src/adapters/poi/idunn_poi.js

```diff
@@ -23,7 +25,7 @@ export default class IdunnPoi extends Poi {
         return
       }
       else {
-        fire('error_h', err)
+        Error.sendOnce('idunn_poi', 'poiApiLoad', 'unknown error getting idunn poi', err)
```

Comment:
Since ajax module does not always provide details about the error, it would be useful to add the url in the message here.